src/style.css

```diff
@@ -0,0 +1,64 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+.bar {
+  display: block;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  box-sizing: border-box;
+  width: 100%;
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+  color: #3d4e61;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-position: 25px;
+  margin: 20px 0;
+  padding: 20px 62px;
```

Comment:
Pay attention to CSS properties order
https://9elements.com/css-rule-order/

Reply:
@polosanya I fixed lines and font style. 
could you advice me with a properties order?